_redirects

```diff
@@ -57,3 +57,6 @@ layout: null
 
 # Without Language
 {{redirectsBase | join: newline}}
+
+# ls was renamed to list
+/en/docs/cli/ls   https://yarnpkg.com/en/docs/cli/list 301
```

Comment:
Maybe do this for every language, and use a relative URI so it works properly at the preview URL:
```
/:language/docs/cli/ls   /:language/docs/cli/list 301
```

Some test cases:
 - https://deploy-preview-377--yarnpkg.netlify.com/en/docs/cli/ls --> should redirect to https://deploy-preview-377--yarnpkg.netlify.com/en/docs/cli/list
 - https://deploy-preview-377--yarnpkg.netlify.com/fr/docs/cli/ls
 - https://deploy-preview-377--yarnpkg.netlify.com/pt-BR/docs/cli/ls